README.txt

```diff
@@ -10,11 +10,11 @@ at https://github.com/z390development/z390/releases
 This distribution has been regression tested on Windows Server and Ubuntu Linux 
 using GitHub Actions.
 
-Java runtime/SDK used: 
-Eclipse Adoptium OpenJDK 8 https://adoptium.net/?variant=openjdk8
+Java runtime/SDK used for build: 
+Eclipse Adoptium OpenJDK 21 https://adoptium.net/?variant=openjdk21
```

Comment:
OpenJDK 21 is the the preferred version for compiling z390. However, to build z390 from source, any Java SDK version 8 or above will do.

Reply:
OK.. I have made a change to the build to use a parameter that only works with Java 9 and above. This is only relevant if you are building. You can run it on Java 8 and above. Hopefully my comments are consistent with this.

Reply:
Okay, I was not aware of the Version 9 requirement. To my mind this should be okay.